operator/cmd/mesh/profile-common.go

```diff
@@ -166,14 +166,26 @@ func genProfile(helmValues bool, inFilename []string, profile, setOverlayYAML, c
 	return finalYAML, err
 }
 
-func unmarshalAndValidateIOP(crYAML string, force bool) (*v1alpha1.IstioOperatorSpec, string, error) {
+func unmarshalAndValidateIOP(crYAML string, force, dumpTranslation bool, l *Logger) (*v1alpha1.IstioOperatorSpec, string, error) {
```

Comment:
it would be more readable to do the translation before passing into this function (since it looks like we're expecting IOP). also dumping could probably be done better outside the function - passing in a bool to dump is a side effect.

Reply:
Fixed.